### ISSUE TITLE
add to pulseaudio

### ISSUE THREAD

**Issue by sandikata** [VladX/ladspa-bass-treble#1]
Привет! Как добавить в Pulseaudio ? Здесь нинакое руководство.

пробовал так pacmd load-module basstreble
но ничего не получилось.
Module load failed.

**Comment by VladX** [comment]
Копируешь плагин в директорию с плагинами ladspa:
`cp basstreble.so /usr/lib/ladspa/`
Дальше в конфиг (например, в _/etc/pulse/default.pa_) добавляешь строчку
`load-module module-ladspa-sink sink_name=ladspa_output.basstreble.bass-treble plugin=basstreble label=bass-treble control=0.4,1.8,0.85`
Модуль будет загружаться вместе с pulseaudio.
Эквалайзер настраивается так: `control=<bass>,<treble>,<amplifier>`

**Comment by sandikata** [comment]
это примерные параметры 0.4,1.8,0.85 если правильно понимаю.

**Comment by VladX** [comment]
Да, для примера. 1 число - уровень низких частот, 2 - уровень высоких частот, 3 - общий уровень. Первые 2 могут быть отрицательными (для ослабления).

**Comment by sandikata** [comment]
Да понятно. А какая то рекомендация для 5.1 surround ? как будет лучше, карточка Asus Xonar DS.

**Comment by VladX** [comment]
Этот плагин для 2 каналов (стерео). Для 5.1 всё по-другому, там нужно отдельный фильтр для сабвуфера и отдельный для всего остального. Этот плагин для такого не предназначен.

**Comment by sandikata** [comment]
Понятно но у меня получилось очень приятно так. Что не нагружает маленкие громкоговорителей когда слышаю на 5.1 вобще хорошая работа, не снижай темпо, улучшай :)

**Comment by sandikata** [comment]
Привет!
После обновление система basstreble более не работает. Pulseaudio вылетает.
E: [alsa-sink-Multichannel] rtpoll.c: Assertion 'p' failed at /var/tmp/portage/media-sound/pulseaudio-8.0/work/pulseaudio-8.0/src/pulsecore/rtpoll.c:613, function pa_rtpoll_item_new_asyncmsgq_write(). Aborting.

Идея какая то ?
